web/src/util.ts

```diff
@@ -489,3 +490,21 @@ export function is_valid_url(url: string, require_absolute = false): boolean {
     }
     return true;
 }
+
+// Formats an array of strings as a Internationalized list using the specified language.
+export function format_array_as_list(
+    array: string[],
+    style: Intl.ListFormatStyle,
+    type: Intl.ListFormatType,
+): string {
+    // If Intl.ListFormat is not supported
+    if (!Intl) {
+        return array.join(", ");
+    }
```

Comment:
Hmm, does this address the bug we saw in 2fc327b775ea4aadec6a47242cef8ef6fce01fce correctly?

I'm not sure if it's possible for `Intl` to exist without `Intl.ListFormat` existing.

Reply:
You're correct; there are situations where Intl may exist without Intl.ListFormat. 
To address this, I've updated the check to verify that Intl.ListFormat is undefined instead of checking for the existence of Intl.